src/explore-education-statistics-common/src/services/permalinkSnapshotService.ts

```diff
@@ -0,0 +1,51 @@
+import { TableJson } from '@common/modules/table-tool/utils/mapTableToJson';
+import { dataApi } from '@common/services/api';
+import { TableDataQuery } from '@common/services/tableBuilderService';
+import { Footnote } from '@common/services/types/footnotes';
+import { UnmappedTableHeadersConfig } from '@common/services/permalinkService';
+
+export interface PermalinkSnapshot {
+  created: string;
+  dataSetTitle: string;
+  id: string;
+  publicationTitle: string;
+  status:
+    | 'Current'
```

Comment:
Would suggest creating a common type for this since it's reused in a couple of places

Reply:
The only place it's reused is in permalinkService which will be replaced by permalinkSnapshotService when the feature flag is removed so doesn't seem necessary to create a common type for it.

Reply:
Ah right, cool. Forgot `permalinkService` on both the BE & FE will be deleted in the future